### PR TITLE
CUMULUS-1581 Add bulk granule support

### PR DESCRIPTION
**Pull request by kkelly51**
- Add "Run Bulk Granules" to Granules page and to Collection pages.

### CODE REVIEW

File: app/src/css/vendor/bootstrap/_components.scss [hunk 1]
```diff
@@ -7,12 +7,14 @@ and behaviors for our Cumulus Dashboard components
...
 
-@import "~bootstrap/scss/_alert";
-@import "~bootstrap/scss/_card";
-@import "~bootstrap/scss/_close";
-@import "~bootstrap/scss/_nav";
-@import "~bootstrap/scss/_popover";
-@import "~bootstrap/scss/_progress";
-@import "~bootstrap/scss/_spinners";
-@import "~bootstrap/scss/_tooltip";
-@import "~bootstrap/scss/_transitions";
```
Comment (jennyhliu) [inline app/src/css/vendor/bootstrap/_components.scss]: Is `~` not working?

File: app/src/js/reducers/granules.js [hunk 2]
```diff
@@ -159,6 +164,30 @@ export default function reducer (state = initialState, action) {
...
+    case BULK_GRANULE:
+    //   data = {
+    //     “createdAt”: 1574730504000,
+    //     “id”: “0eb8e809-8790-5409-1239-bcd9e8d28b8e”,
+    //     “updatedAt”: 1574730504762,
+    //     “status”: “RUNNING”,
+    //     “taskArn”: “arn:aws:ecs:us-east-1:111111111111:task/d481e76e-f5fc-9c1c-2411-fa13779b111a”
+    // }
+      // console.log('in case granule_BULK', data);
+      // console.log('id in reducer', requestId);
```
Comment (chuckwondo) [inline app/src/js/reducers/granules.js]: Any reason to keep the code commented out above? Can we just remove it?

Comment (kkelly51) [reply app/src/js/reducers/granules.js]: ahh good catch, I will remove

File: app/src/css/vendor/bootstrap/_components.scss (hunk 1 above)
Comment (jennyhliu) [reply app/src/css/vendor/bootstrap/_components.scss]: Is end of file `new line` not allowed in .scss?

Comment (kkelly51) [reply app/src/css/vendor/bootstrap/_components.scss]: @jjmccoy  ??

**Pull request merged**